src/index.js

```diff
@@ -3,26 +3,23 @@ import './imgs/enter.png';
 import './imgs/menu.png';
```

Comment:
- [ ] Please add a new javascript file for the new functionality which is to add and remove tasks as required in the instructions 👍 .
- [ ] When I add a task and refresh the page, it returns an undefined task. Please fix that.
![image](https://user-images.githubusercontent.com/45460192/193263265-3d85a9e9-92f4-472c-bf1b-06bc14c3168a.png)
- [ ] When I delete an item and refresh the page, it just returns the item as checked and remains on the local storage.

Reply:
sorry but im stuck right there i don't know how to do it can you help me